devices/YarpRobotLoggerDevice/include/BipedalLocomotion/YarpRobotLoggerDevice.h

```diff
@@ -191,8 +202,59 @@ class YarpRobotLoggerDevice : public yarp::dev::DeviceDriver,
     bool createFramesFolder(std::shared_ptr<VideoWriter::ImageSaver> imageSaver,
                             const std::string& camera,
                             const std::string& imageType);
+
+    const std::string treeDelim = "::";
+
+    const std::string robotRtRootName = "robot_realtime";
+
+    const std::string jointStatePositionsName = "joints_state::positions";
+    const std::string jointStateVelocitiesName = "joints_state::velocities";
+    const std::string jointStateAccelerationsName = "joints_state::accelerations";
+    const std::string jointStateTorquesName = "joints_state::torques";
+
+    const std::string motorStatePositionsName = "motors_state::positions";
+    const std::string motorStateVelocitiesName = "motors_state::velocities";
+    const std::string motorStateAccelerationsName = "motors_state::accelerations";
+    const std::string motorStateCurrentsName = "motors_state::currents";
+    const std::string motorStatePwmName = "motors_state::PWM";
+
+    const std::string motorStatePidsName = "PIDs";
+
+    const std::string ftsName = "FTs";
+
+    const std::vector<std::string> ftElementNames = {"f_x", "f_y", "f_z", "mu_x", "mu_y", "mu_z"};
+
+    const std::string gyrosName = "gyros";
+    const std::vector<std::string> gyroElementNames = {"omega_x", "omega_y", "omega_z"};
+
+    const std::string accelerometersName = "accelerometers";
+    const std::vector<std::string> AccelerometerElementNames = {"a_x", "a_y", "a_z"};
+
+    const std::string orientationsName = "orientations";
+    const std::vector<std::string> orientationElementNames = {"r", "p", "y"};
+
+    const std::string magnetometersName = "magnetometers";
+    const std::vector<std::string> magnetometerElementNames = {"mag_x", "mag_y", "mag_z"};
+
+    const std::string cartesianWrenchesName = "cartesian_wrenches";
+    const std::vector<std::string> cartesianWrenchNames = {ftElementNames[0],
+                                                        ftElementNames[1],
+                                                        ftElementNames[2],
+                                                        ftElementNames[3],
+                                                        ftElementNames[4],
+                                                        ftElementNames[5]};
```

Comment:
constexpr initialized the variables at compile time. This is not necessary in this case, these variables can be initialized during run time because the functions which use them will be called after they complete their initialization. Additionally, C++ currently does not allow a constexpr of a vector or of std::string objects.